package.json

```
@@ -2,11 +2,37 @@
   "name": "trunkio-plugins",
   "version": "1.0.0",
   "scripts": {
-    "trunk": "trunk"
+    "trunk": "trunk",
+    "build-test": "tsc",
+    "test": "jest"
```

I recommend a hook for `test` to always `build-test` prior:

```suggestion
    "build-test": "tsc",
    "pretest": "npm run build-test",
    "test": "jest"
```